examples/qt_framework/qimage_display/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Find includes in the build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Turn on automatic invocation of the MOC
+set(CMAKE_AUTOMOC ON)
+
+# Add a compiler flag
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wstrict-aliasing -Wpedantic -Wconversion -Wsign-conversion")
+
+# Find the QtWidgets library
+find_package(Qt5Widgets REQUIRED)
+set(TARGET_NAME qimage_display)
+
+# Generate code from ui files => qt5_wrap_ui
+# Generate rules for building source files from the resources => qt5_add_resources
+
+set(LIB_DIR ${CMAKE_SOURCE_DIR}/src)
+set(SRCS
+    ${LIB_DIR}/blob_detection.cpp
+    ${LIB_DIR}/image_function.cpp
```

Comment:
Please include `${LIB_DIR}/penguinv/penguinv.cpp` file as well due to latest changes.